frontend/src/lib/components/PropertyFilters/components/FilterRow.tsx

```diff
@@ -96,9 +100,7 @@ export const FilterRow = React.memo(function FilterRow({
                         )}
                     </Popover>
                 )}
-                {key && showConditionBadge && index + 1 < totalCount && (
-                    <span className="stateful-badge and ml-2">AND</span>
-                )}
+                {key && showConditionBadge && index + 1 < totalCount && <LemonTag type="highlight">AND</LemonTag>}
```

Comment:
Flyby comment: Should we create a wrapper component for the AND and OR tags? We likely want them to have the same colors throughout the app.